UnitTests/BraintreeThreeDSecureTests/BTThreeDSecureClient_Tests.swift

```diff
@@ -350,30 +348,55 @@ class BTThreeDSecureClient_Tests: XCTestCase {
             XCTAssertFalse(tokenizedCard.threeDSecureInfo.liabilityShiftPossible)
             XCTAssertTrue(tokenizedCard.threeDSecureInfo.wasVerified)
             XCTAssertNil(error)
-            expectation.fulfill()
         }
 
-        waitForExpectations(timeout: 4)
+        waitForExpectations(timeout: 10)
         XCTAssertTrue(mockAPIClient.postedAnalyticsEvents.contains(BTThreeDSecureAnalytics.verifySucceeded))
     }
 
-    // TODO: - this test doesn't do what it says it does
     func testStartPayment_v2_callsOnLookupCompleteDelegateMethod() {
         threeDSecureRequest.threeDSecureRequestDelegate = mockThreeDSecureRequestDelegate
-
-        let expectation = expectation(description: "willCallCompletion")
-
+        mockThreeDSecureRequestDelegate.lookupCompleteExpectation = expectation(description: "Lookup completed successfully")
         mockAPIClient.cannedConfigurationResponseBody = BTJSON(value: [
             "threeDSecure": ["cardinalAuthenticationJWT": "FAKE_JWT"],
             "assetsUrl": "http://assets.example.com"
         ] as [String: Any])
-        let client = BTThreeDSecureClient(apiClient: mockAPIClient)
 
-        client.startPaymentFlow(threeDSecureRequest) { result, error in
-            expectation.fulfill()
-        }
+        let responseBody = [
+            "paymentMethod": [
+                "consumed": false,
+                "description": "ending in 02",
+                "details": [
+                    "cardType": "Visa",
+                    "lastTwo": "02",
+                ],
+                "nonce": "f689056d-aee1-421e-9d10-f2c9b34d4d6f",
+                "threeDSecureInfo": [
+                    "enrolled": "Y",
+                    "liabilityShiftPossible": true,
+                    "liabilityShifted": true,
+                    "status": "authenticate_successful",
+                ] as [String: Any],
+                "type": "CreditCard",
+            ] as [String: Any],
+            "success": true,
+            "threeDSecureInfo":     [
+                "liabilityShiftPossible": true,
+                "liabilityShifted": true,
+            ],
+            "lookup": [
```

Comment:
This specific test was looking for the existence of this `lookup` object. Mocking this result allows us to speed up and hopefully hit this timeout less frequently.